greentest/2.7/test_ssl.py

```diff
@@ -107,17 +126,19 @@ def test_parse_cert(self):
         # note that this uses an 'unofficial' function in _ssl.c,
         # provided solely for this test, to exercise the certificate
         # parsing code
-        p = ssl._ssl._test_decode_cert(CERTFILE, False)
+        try:
+            p = ssl._ssl._test_decode_cert(CERTFILE, False)
```

Comment:
It's called 'verbose', true, but in <= 2.7.8 it actually activates an additional set of tests in decoding the certificate. These tests are enabled by default in 2.7.9. But because of the different certificates and the possible different OpenSSL configurations, I wasn't sure whether it might cause problems. It seemed safer to me just to leave it.